Elevel/Application/Features/TopicCommands/CreateTopicCommand.cs

```diff
@@ -10,19 +12,13 @@ namespace Elevel.Application.Features.TopicCommands
 {
     public class CreateTopicCommand
     {
-        public class Request : UpsertTopicCommand.Request, IRequest<Response>
+        public class Request : IRequest<Response>
         {
-
+            public string TopicName {get; set;}
+            public Level Level { get; set; }
+            public Guid CreatorId { get; set; }
```

Comment:
it should not be in Request, server should populate it, without property from Client